solutions/swb-reference/src/postDeployment/authorizationSetup.ts

```diff
@@ -24,34 +25,34 @@ export default class AuthorizationSetup {
   }
 
   public async run(): Promise<void> {
-    const groupName = 'ITAdmin';
-    const adminUser: AuthenticatedUser = { id: this._constants.ROOT_USER_EMAIL, roles: [groupName] };
+    const itAdmin = 'ITAdmin';
+    const adminUser: AuthenticatedUser = { id: this._constants.ROOT_USER_EMAIL, roles: [itAdmin] };
 
-    await this.createGroupIfNotExist(groupName, 'IT Admin group for SWB.', adminUser);
+    await this.createGroupIfNotExist(itAdmin, 'IT Admin group for SWB.', adminUser);
 
-    const projectPermissions = this._mapActions(groupName, 'Project', ['User']);
-    const environmentTypePermissions = this._mapActions(groupName, 'EnvType', ['EnvTypeConfig']);
-    const datasetPermissions = this._mapActions(groupName, 'ExternalDataset');
-
-    const userPermissions = this._mapActions(groupName, 'User');
-    const costCenterPermissions = this._mapActions(groupName, 'CostCenter');
-    const awsAccountPermissions = this._mapActions(groupName, 'AwsAccount');
-    const groupPermissions = this._mapActions(groupName, 'Group | groupId', [], ['UPDATE']);
+    //IT Admin Permissions
+    const projectPermissions = this._mapActions(itAdmin, SwbAuthZSubject.SWB_PROJECT);
+    const userPermissions = this._mapActions(itAdmin, SwbAuthZSubject.SWB_USER);
+    const environmentTypePermissions = this._mapActions(itAdmin, SwbAuthZSubject.SWB_ENVIRONMENT_TYPE);
+    const environmentTypeConfigPermissions = this._mapActions(itAdmin, SwbAuthZSubject.SWB_ETC);
+    const datasetPermissions = this._mapActions(itAdmin, SwbAuthZSubject.SWB_DATASET);
+    const costCenterPermissions = this._mapActions(itAdmin, SwbAuthZSubject.SWB_COST_CENTER);
+    const awsAccountPermissions = this._mapActions(itAdmin, SwbAuthZSubject.SWB_AWS_ACCOUNT);
 
```

Comment:
Add an environment * permission that allows ITAdmin to READ, UPDATE, and DELETE.